.vsts-ci/mac.yml

```diff
@@ -91,16 +91,17 @@ stages:
     parameters:
       pool: macOS-latest
 
-- stage: CodeCovTestPackage
-  displayName: CodeCoverage and Test Packages
-  dependsOn: [] # by specifying an empty array, this stage doesn't depend on the stage before it
-  jobs:
-  - job: CodeCovTestPackage
-    displayName: CodeCoverage and Test Packages
-    pool:
-      vmImage: macOS-latest
-    steps:
-    - pwsh: |
-        Import-Module .\tools\ci.psm1
-        New-CodeCoverageAndTestPackage
-      displayName: CodeCoverage and Test Package
+# this is no longer used
```

Comment:
Can we just delete this? We have history if we need to bring this back.

Reply:
Done.